streams/src/main/java/org/apache/kafka/streams/state/internals/MemoryLRUCache.java

```diff
@@ -74,7 +74,7 @@ public MemoryLRUCache(String name, final int maxCacheSize, Serde<K> keySerde, Se
 
             @Override
             protected boolean removeEldestEntry(Map.Entry<K, V> eldest) {
-                if (size() > maxCacheSize) {
+                if (super.size() > maxCacheSize) {
```

Comment:
Findbugs warned about this potentially being ambiguous, since there is an inherited method size here, and a method in the outer (enclosing) class.  As it turns out, it's not an issue here since both of these methods return the same thing.  But this makes it explicit which one we are calling.